lib/workers/repository/process/write.ts

```diff
@@ -35,7 +42,26 @@ export async function writeUpdates(
   for (const branch of branches) {
     addMeta({ branch: branch.branchName });
     const branchExisted = branchExists(branch.branchName);
-    const res = await processBranch(branch);
+    let branchCache = cachedBranches?.find(
+      (br) => br.branchName === branch.branchName
+    );
+
+    if (branchCache === undefined) {
+      if (config.repositoryCache === 'enabled') {
+        logger.debug(`No branch cache found for ${branch.branchName}`);
+      }
```

Comment:
Not sure it makes sense to log this if branchExisted=false? If branch doesn't exist then it should be normal case to have no branch cache?